PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,7 +3,12 @@ Checklist:
 <!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->
 
 - [ ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
-- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
+- [ ] I have read [freeCodeCamp's guidelines for creating a Pull Request](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
+- [ ] My pull request has a descriptive title and is less than 30 characters. Good examples are: 
+  - `feat: add more tests to HTML and CSS challenges`
+  - `fix(a11y): improved search bar contrast`
+  - `fix(api,client): prevent CORS errors on form submission`
+     
```

Comment:
```suggestion
- [ ] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)
```

Reply:
Thanks for taking the time to review this and make a suggestion, LGTM!